server/store/api.go

```diff
@@ -13,6 +14,10 @@ type API interface {
 	KVGet(key string) ([]byte, *model.AppError)
 	KVSet(key string, value []byte) *model.AppError
 	KVDelete(key string) *model.AppError
+	LogDebug(msg string, keyValuePairs ...interface{})
+	LogInfo(msg string, keyValuePairs ...interface{})
+	LogError(msg string, keyValuePairs ...interface{})
+	LogWarn(msg string, keyValuePairs ...interface{})
```

Comment:
Which methods does use them?

Reply:
logError is used to log the error in some function , so had a thought to implement all the function which might get used in future